actors/common/src/main/java/org/sunbird/metrics/actors/OrganisationMetricsActor.java

```diff
@@ -527,22 +542,24 @@ private void orgConsumptionMetrics(Request actorMessage) {
         return;
       }
       String channel = (String) rootOrgData.get(JsonKey.HASHTAGID);
-      ProjectLogger.log("channel" + channel);
-      String responseFormat = (String) cache.getData(JsonKey.OrgConsumption, orgId, periodStr);
-      if (StringUtils.isBlank(responseFormat)) {
-        responseFormat = getOrgConsumptionData(actorMessage, periodStr, orgHashId, channel);
-        ProjectLogger.log("Response" + responseFormat);
-        cache.setData(JsonKey.OrgConsumption, orgId, periodStr, responseFormat);
-      }
+      ProjectLogger.log(
+          "OrganisationMetricsActor:orgConsumptionMetrics hash tagid value " + channel,
```

Comment:
From:
"OrganisationMetricsActor:orgConsumptionMetrics hash tagid value "

To:
"OrganisationMetricsActor:orgConsumptionMetrics: hash tag id = "